CONTRIBUTING.md

```diff
@@ -1,43 +1,39 @@
-# How to Contribute to asdf
+# How to Contribute to `asdf`
```

Comment:
Why is asdf in backticks? This seems inconsistent with some exist content on the documentation site.

Reply:
I removed them from this page. I tend to ref `asdf` the name in backticks in most places. I have done so on many pages throughout the docs. Perhaps we can discuss the correct usage after the new site is live.

Reply:
I guess I prefer asdf when referring to the whole project, and only using `asdf` when specifically referring to the command-line executable.